Terraform 1.1 requires upgrade of `terraform-json` for acceptance tests

### SDK version
<!--
Inspect your go.mod as below to find the version, and paste the result between the ``` marks below.

go list -m github.com/hashicorp/terraform-plugin-sdk/...

If you are not running the latest version of the SDK, please try upgrading
because your feature may have already been implemented.

If the command above doesn't yield any results, it means you may either be using v1 of the SDK or
have not have migrated to the standalone SDK yet. See https://www.terraform.io/docs/extend/plugin-sdk.html
for more.
-->
```
github.com/hashicorp/terraform-plugin-sdk/v2 v2.8.0
```

### Use-cases

Running [Terraform CLI v1.1.0-beta1](https://github.com/hashicorp/terraform/blob/v1.1.0-beta1/CHANGELOG.md) with the latest AWS Provider acceptance tests results in every test failing:

```
=== RUN   TestAccAccessAnalyzer_serial/Analyzer/basic
analyzer_test.go:24: Step 1/2 error: unsupported state format version: expected ["0.1" "0.2"], got "1.0"
testing_new.go:63: Error retrieving state, there may be dangling resources: unsupported state format version: expected ["0.1" "0.2"], got "1.0"
```

The [`terraform-json`](https://github.com/hashicorp/terraform-json) dependency needs to be upgraded to at least [**v0.13.0**](https://github.com/hashicorp/terraform-json/releases/tag/v0.13.0).

Relates: https://github.com/hashicorp/terraform-json/pull/41.

--- a/go.mod
+++ b/go.mod
@@ -20,8 +20,8 @@ require (
 	github.com/hashicorp/go-version v1.3.0
 	github.com/hashicorp/hcl/v2 v2.3.0
 	github.com/hashicorp/logutils v1.0.0
-	github.com/hashicorp/terraform-exec v0.14.0
-	github.com/hashicorp/terraform-json v0.12.0
+	github.com/hashicorp/terraform-exec v0.15.0
+	github.com/hashicorp/terraform-json v0.13.0
 	github.com/hashicorp/terraform-plugin-go v0.4.0
 	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
 	github.com/kylelemons/godebug v1.1.0 // indirect
@@ -30,7 +30,7 @@ require (
 	github.com/mitchellh/go-wordwrap v1.0.0 // indirect
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/mitchellh/reflectwalk v1.0.2
-	github.com/zclconf/go-cty v1.8.4
+	github.com/zclconf/go-cty v1.9.1
 	golang.org/x/crypto v0.0.0-20210421170649-83a5a9bb288b
 	golang.org/x/tools v0.0.0-20200713011307-fd294ab11aed
 	google.golang.org/grpc v1.32.0

